Form-Controls/styles.css

```diff
@@ -0,0 +1,106 @@
+/* Reset some default styles */
```

Comment:
Great job on styling the CCS. It looks really good but I'm getting Lighthouse Accessibility score below 100 because of "Background and foreground colours not having a sufficient contrast ratio". How could you improve this?